Gems/RobotecSplineTools/Code/Source/Tools/SplineToolsEditorComponent.h

```diff
@@ -43,5 +43,7 @@ namespace SplineTools
 
         void ReloadCSVAsset();
         AZStd::vector<AZ::Vector3> GetSplinePointsFromCsv(const AZStd::string& csvFilePath);
+        void SaveCsvAsset();
+        bool SetSplinePointsToCsv(const AZStd::string& csvFilePath, const AZStd::vector<AZ::Vector3>& vertices);
```

Comment:
Maybe `SaveSplinePointsToCsv` would be a better name?

Reply:
Changed to `SavePointsToCsv`